internal/xds/translator/authentication.go

```diff
@@ -195,34 +195,25 @@ func buildClusterFromJwks(jwks *jwksCluster) (*cluster.Cluster, error) {
 		return nil, errors.New("jwks is nil")
 	}
 
-	endpoints, err := jwks.toLbEndpoints()
+	destinations, err := jwks.toRouteDestinations()
 	if err != nil {
 		return nil, err
 	}
 
+	cluster := buildXdsCluster(jwks.name, destinations, false /*isHTTP2 */)
+	cluster.ConnectTimeout = durationpb.New(10 * time.Second)
+	cluster.DnsRefreshRate = durationpb.New(30 * time.Second)
+	cluster.LbPolicy = clusterv3.Cluster_RANDOM
+	cluster.RespectDnsTtl = true
```

Comment:
I believe RespectDnsTtl can be dropped since the cluster is static.

As a follow-on PR, it would be nice to have a type that represents cluster config params, e.g. connection timeout, destinations, etc., that can be instantiated and passed into buildXdsCluster().